README.md

```diff
@@ -28,6 +28,13 @@ Compilation
 
 Cloud Gateway's build is based on `cmake`.
 
+
+Install prerequisites:
+
+```
+sudo apt-get install libz-dev libssl-dev
```

Comment:
so, why are those not included in the prerequisites above?

Reply:
They are not needed to build libevaio. But we can put them with the previous one.

Reply:
Ah, it was not clear to me that the previous ones were only related to `libevaio`. I think it would be easier to group them, yes.

Reply:
done